steps/pytorch/callbacks.py

```diff
@@ -367,6 +367,105 @@ def on_batch_begin(self, *args, **kwargs):
 
 
 class ReduceLROnPlateau(Callback):  # thank you keras
-    def __init__(self):
+    def __init__(self, lr_factor, lr_patience, epoch_every=1, batch_every=None):
         super().__init__()
-        pass
+        self.factor = lr_factor
+        self.patience = lr_patience
+        if epoch_every == 0:
+            self.epoch_every = False
```

Comment:
@apyskir 0 is treated as False in all usual cases so I am not sure if we needed that.

Reply:
I just copy-pasted it from other places like https://github.com/neptune-ml/open-solution-data-science-bowl-2018/blob/dev-solution-5/steps/pytorch/callbacks.py#L105 or https://github.com/neptune-ml/open-solution-data-science-bowl-2018/blob/dev-solution-5/steps/pytorch/callbacks.py#L144, but I agree - we don't need it. I should have corrected that.